mscore/inspector/inspector.cpp

```diff
@@ -577,6 +591,20 @@ InspectorFermata::InspectorFermata(QWidget* parent)
       mapSignals(iiList, ppList);
       }
 
+//---------------------------------------------------------
+//   setElement
+//---------------------------------------------------------
+
+void InspectorFermata::setElement()
+      {
+      InspectorElementBase::setElement();
+      if (!f.playArticulation->isChecked()) {
+            f.label_2->setEnabled(false);
```

Comment:
Could we perhaps rename those `label_2` widgets in `.ui` files to something more sensible? Then this code would become more clear.

Reply:
Will do! Thanks!